tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature

```diff
@@ -84,17 +84,16 @@ Feature: Create public link shares
     And the public uses the webUI to access the last public link created by user "Alice"
     Then file "lorem.txt" should be listed on the webUI
 
-  @skip @yetToImplement
+  @issue-5302
   Scenario: share two file with same name but different paths by public link
     Given user "Alice" has created folder "simple-folder"
     And user "Alice" has created file "simple-folder/lorem.txt"
     And user "Alice" has created file "lorem.txt"
     And user "Alice" has logged in using the webUI
     When the user creates a new public link for file "lorem.txt" using the webUI
-    And the user closes the details dialog
     And the user opens folder "simple-folder" using the webUI
     And the user creates a new public link for file "lorem.txt" using the webUI
-    And the user browses to the shared-by-link page
+    And the user has browsed to the shared-via-link page
```

Comment:
Since this is part of `when` step I think `browses` is correct

Reply:
```suggestion
    And the user browses to the shared-via-link page
```
This changed line can be reverted. (I don't know if "via" is now the correct step text?)

Reply:
yeah, there is `shared-via-link` but not `shared-by-link`.